docs/kiosk-data/GameList.json

```diff
@@ -1,4 +1,5 @@
 {
+    "TODO: Delete this file after the next Arcade release": 1,
```

Comment:
I'm guessing you can't delete this now because it's still being used?

Reply:
Changes to the docs folder take affect almost immediately, and that can impact the current release. The released version of kiosk doesn't reference this file, so it would be fine. The beta version does though. Not the end of the world. Bumping beta would fix that.